src/crypto-api/CrossSigningInfo.ts

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { CrossSigningKey } from "../crypto-api";
+
+/**
+ * Information about a user's cross-signing keys
+ */
+export interface CrossSigningInfo {
```

Comment:
I would have put this in `crypto-api.ts`, but if you're going to put it here, please re-export it from `crypto-api.ts` so that it is correctly documented at https://pr3646--js-sdk-docs-previews.netlify.app/stable/modules/crypto.

Reply:
Also: can we find a name for it that distinguishes it from [`CrossSigningKeyInfo`](https://github.com/matrix-org/matrix-js-sdk/blob/0a69f6ecc540b8ff58333c487d5ad684c793ddb8/src/crypto-api.ts#L583) ?